### PR TITLE
Update ACM channel to release-2.7

### PR DESCRIPTION
**Pull request by sidhant-agrawal**
Fixes: #7495

### CODE REVIEW

**Review by keesturam — DISMISSED** [review]

**Review by am-agrawa — DISMISSED** [review]

**Review by prsurve — DISMISSED** [review]

**Review by petr-balogh — APPROVED** [review]

**Comment by petr-balogh** [comment]
But I guess it's worth to verify the 4.12 deployment with ACM 2.7, @shylesh do you think it will require some changes from automation of deployment as well?

File: ocs_ci/framework/conf/ocp_version/ocp-4.10-config.yaml [hunk 1]
```diff
@@ -10,4 +10,4 @@ DEPLOYMENT:
   ignition_version: "3.2.0"
 ENV_DATA:
   vm_template: "rhcos-410.84.202201251210-0-vmware.x86_64"
-  acm_hub_channel: release-2.5 # To be changed when acm 2.6 get's released
+  acm_hub_channel: release-2.7 # To be changed when acm 2.8 get's released
```
[the remark this answers is not in the document]
Comment (shylesh) [reply ocs_ci/framework/conf/ocp_version/ocp-4.10-config.yaml]: @petr-balogh , Yes, 2.5 is for 4.11 and older, from 4.12 onward we have been sticking to 2.7

**Comment by shylesh** [comment]
> 

@petr-balogh , I have already verified this earlier, may be after must-gather multicluster changes, I will be backporting and testing with 4.12 + 2.7

**Review by shylesh — APPROVED** [review]
+1

**Review by keesturam — APPROVED** [review]

**Pull request merged**